Code/Framework/AzCore/AzCore/Memory/MallocSchema.cpp

```diff
@@ -144,6 +144,11 @@ AZ::MallocSchema::size_type AZ::MallocSchema::GetMaxAllocationSize() const
     return 0xFFFFFFFFull;
 }
 
+AZ::MallocSchema::size_type AZ::MallocSchema::GetMaxContiguousAllocationSize() const
+{
+    return 0xFFFFFFFFull;
```

Comment:
here we should also return `AZ_CORE_MAX_ALLOCATOR_SIZE`

This is using `malloc` under the hood, according to [this](https://stackoverflow.com/questions/2798330/maximum-memory-which-malloc-can-allocate), SIZE_MAX is the maximum possible request.

Reply:
Will update